packages/expo-web-browser/ios/WebBrowserModule.swift

```diff
@@ -8,17 +8,27 @@ final public class WebBrowserModule: Module {
   private var currentWebBrowserSession: WebBrowserSession?
   private var currentAuthSession: WebAuthSession?
 
+  private func isValid(url: URL) -> Bool {
+    return url.scheme == "http" || url.scheme == "https"
+  }
```

Comment:
Next time, put functions like this outside of the class – it is not depending on the class instance in any way 😉

Reply:
🥶

Should I create a WebBrowserUtils.swift and move it into its own file?

Reply:
I mean the check is even so simple I could just inline it

Reply:
Inlining would be fine too, but it can be in the same file as it doesn't need to be accessible from other files 😉 In utils files you usually want them to be exposed (`public` or `internal`).

Reply:
Make sense.